pdf/core/security/handlers.go

```diff
@@ -30,3 +32,22 @@ type StdEncryptDict struct {
 	OE, UE []byte // R=6
 	Perms  []byte // An encrypted copy of P (16 bytes). Used to verify permissions. R=6
 }
+
+// assertSize checks the size of the bytes field and returns a descriptive error if it doesn't.
+func assertSize(fnc, field string, exp int, b []byte) error {
+	if len(b) < exp {
```

Comment:
Perhaps the name should reflect that it is not an exact match.
Like `checkAtLeast` checks that `b` is at least `exp` bytes and returns a descriptive error if it doesn't.